### ISSUE TITLE
Missing associated icns (v7.0.3, OS X)

### ISSUE THREAD

**Issue by diegohs77** [teras/Jubler#23]
Since the v7.0.3 there's no "subtitle.icns" and it's association. I don't know if it's an issue or it's deliberate.
Thanks for the program!!!!! Cheers!

**Comment by diegohs77** [comment]
Another thing I just discover is QLAddict.qlgenerator no longer works with 7.0.3.

**Comment by teras** [comment]
It works with me. Maybe it's an issue with your installation? You might have installed something?

**Comment by diegohs77** [comment]
No, just installing 7.0.3 breaks QL. I tried resetting the QL and no luck. Deleting 7.0.3 and using again v7.0.0 solve this issue for me (and the .icns preview ;) ). Thanks anyway teras !!!
